apps/client/src/routes/dashboards/dashboards-index/hooks/use-delete-dashboard-mutation.ts

```diff
@@ -1,12 +1,10 @@
 import { useMutation } from '@tanstack/react-query';
-import { deleteDashboard } from '~/services';
-import { invalidateDashboard } from '~/data/dashboards';
-
-import type { DashboardSummary } from '~/services';
+import { bulkDeleteDashboards } from '~/services';
+import { invalidateDashboards } from '~/data/dashboards';
 
 export function useDeleteDashboardMutation() {
   return useMutation({
-    mutationFn: (dashboard: DashboardSummary) => deleteDashboard(dashboard.id),
-    onSuccess: (_data, variables) => void invalidateDashboard(variables.id),
+    mutationFn: bulkDeleteDashboards,
+    onSuccess: () => void invalidateDashboards(),
```

Comment:
I'll need to check this out. My thinking was it didn't matter, since this would cause the table page to refresh. I'll need to check if the dashboard page is still accessible. If it is, I'll need to update the logic.

Reply:
We're good. The only way to access the dashboard page after deletion is through the "create dashboard" success notification, which may still be visible when the dashboard is deleted. The single dashboard cache only hydrates when visiting the dashboard page.

The "create dashboard" success notification disappears once you leave the dashboard list page. Although the "update dashboard" function calls `GET /dashboards/{dashboardId}` during inline table editing, it doesn't hydrate the cache. This means that even if you try to access the dashboard page via the success notification after deleting it, you'll be directed to a 404 error page.